app/views/sites/index.html.erb

```diff
@@ -49,7 +49,7 @@
         <%= render @sites %>
       </tbody>
     </table>
-    <%= add_location_link(current_user['info']['email'], @provider) %>
+    <%= link_to "Add a location", new_provider_site_path(@provider.provider_code), class: "govuk-button govuk-!-margin-bottom-2" %>
     <p class="govuk-body-s govuk-!-margin-bottom-6">You’ll be taken to a Google Form to fill in your location details.</p>
```

Comment:
This text should be removed when the button is changed.

Reply:
Done